addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -0,0 +1,208 @@
+                    text: $(this).attr('text_value'),
+                    is_correct: $(this).attr('is_correct')
+                })
+            })
+        },
+        _updateProgressbar: function(){
+            var self = this;
+            var completion = self.channelCompletion <= 100 ? self.channelCompletion : 100;
+            $('.o_wslides_fullscreen_sidebar_progress_gauge').css('width', completion + "%" )
+            $('.o_wslides_progress_percentage').text(completion);
+        },
+        _bindQuizEvents: function(){
+            var self = this;
+            if(!self.slide.done){
+                $('.o_wslides_quiz_answer').each(function(){
+                    $(this).click(self._onAnswerClick.bind(self))
+                })
+            }
+            $('.submit-quiz').click(self._onSubmitQuiz.bind(self))
+        },
+        _highlightAnswers: function(answers){
+            var self = this;
+            self.answeredQuestions = [];
+            for(var i = 0; i < answers.goodAnswers.length; i++){
+                $('#answer'+ answers.goodAnswers[i] +'').addClass('o_wslides_quiz_good_answer');
+                $('#answer'+ answers.goodAnswers[i] +' .o_wslides_quiz_radio_box span').replaceWith($('<i class="fa fa-check-circle"></i>'))
+                var questionID =$('#answer'+ answers.goodAnswers[i] +' .o_wslides_quiz_radio_box input').attr('question_id');
+                self.answeredQuestions.push(questionID);
+                $('.o_wslides_quiz_answer[question_id='+questionID+']:not(.o_wslides_quiz_good_answer)').addClass('o_wslides_quiz_ignored_answer');
+                $('.o_wslides_quiz_answer[question_id='+questionID+']').each(function(){
+                    $(this).unbind('click')
+                });
+                $('input[question_id='+questionID+']').each(function(){
+                    $(this).prop('disabled',true);
+                })
+            }
+            for(var i = 0; i < answers.badAnswers.length; i++){
+                $('#answer'+ answers.badAnswers[i]).removeClass('o_wslides_quiz_good_answer')
+                    .addClass('o_wslides_quiz_bad_answer')
+                    .unbind('click');
+                $('#answer'+ answers.badAnswers[i] +' .o_wslides_quiz_radio_box span').replaceWith($('<i class="fa fa-times "></i>'));
+                $('#answer'+ answers.badAnswers[i] +' .o_wslides_quiz_radio_box input').prop('checked', false);
+            }
+        },
+        //--------------------------------------------------------------------------
+        // Handlers
+        //--------------------------------------------------------------------------
+        _onAnswerClick: function(ev){
+            var self = this;
+            var target = $(ev.currentTarget)
+            if((self.answeredQuestions.indexOf(target.attr('question_id'))) === -1){
+                var id = target.attr('id')
+                var question_id = target.attr('question_id')
```

Comment:
questionId, and as already said should use target.data('question_id') when possible (and change to t-att-data-question_id on xml side accordingly).